frontend/public/locales/en/chat.json

```diff
@@ -12,7 +12,7 @@
     "receivedResponse": "Received response. Starting to handle stream...",
     "errorCallingAPI": "Error calling the API",
     "ask": "Ask a question, or describe a task.",
-    "begin_conversation_placeholder": "Begin conversation here...",
+    "begin_conversation_placeholder": "Begin conversation here, @ to select a brain or / to select a file",
```

Comment:
The used one seems to be `actions_bar_placeholder` instead of `begin_conversation_placeholder`

Reply:
It is not

Reply:
Fixed